vm-setup/roles/v1aX_integration_test/tasks/move_cleanup.yml

```diff
@@ -0,0 +1,108 @@
+---
+  - name: Deprovision Ubuntu based worker node
+    block:
+      - name: Copy generated controlplane_ubuntu.yaml
+        copy:
+          src: "{{ TEMP_GEN_DIR }}/{{ CAPI_VERSION }}_workers_ubuntu.yaml"
+          dest: "/tmp/workers_ubuntu.yaml"
+
+      - name: Deprovision Ubuntu worker node.
+        k8s:
+          state: absent
+          src: "/tmp/workers_ubuntu.yaml"
+          namespace: "{{ NAMESPACE }}"
+        environment:
+          KUBECONFIG: "/tmp/target.yaml"
+        ignore_errors: yes        
+    when: IMAGE_OS == "Ubuntu"
+
+  - name: Deprovision CentOS based worker node
+    block:
+      - name: Copy generated controlplane_ubuntu.yaml
+        copy:
+          src: "{{ TEMP_GEN_DIR }}/{{ CAPI_VERSION }}_workers_centos.yaml"
+          dest: "/tmp/workers_centos.yaml"
+
+      - name: Deprovision CentOS worker node.
+        k8s:
+          state: absent
+          src: "/tmp/workers_centos.yaml"
+          namespace: "{{ NAMESPACE }}"
+        ignore_errors: yes
+        environment:
+          KUBECONFIG: "/tmp/target.yaml"
+    when: (IMAGE_OS == "Centos") or
+          (IMAGE_OS == "")
+
+  - name: Deprovision Ubuntu based controlplane node
+    block:
+      - name: Copy generated controlplane_ubuntu.yaml
+        copy:
+          src: "{{ TEMP_GEN_DIR }}/{{ CAPI_VERSION }}_controlplane_ubuntu.yaml"
+          dest: "/tmp/controlplane_ubuntu.yaml"
+
+      - name: Deprovision Ubuntu master node.
+        k8s:
+          state: absent
+          src: "/tmp/controlplane_ubuntu.yaml"
+          namespace: "{{ NAMESPACE }}"
+        environment:
+          KUBECONFIG: "/tmp/target.yaml"
+        ignore_errors: yes
+    when: IMAGE_OS == "Ubuntu"
```

Comment:
Can this work without pivoting back to the ephemeral cluster ?

Reply:
It starts deprovisioning but its too fast to catch at times. I have confirmed it since the nodes restart.  I am just keeping it there for future references